ReactViewControl.Avalonia/ReactViewControl.Avalonia.csproj

```diff
@@ -38,7 +38,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="WebViewControl-Avalonia" Version="2.91.23" />
+    <PackageReference Include="WebViewControl-Avalonia" Version="2.106.0">
+      <IncludeAssets>All</IncludeAssets>
+      <PrivateAssets>None</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="Avalonia.Desktop" Version="0.10.17" />
```

Comment:
nope... this isnt needed

Reply:
We had to add that because of these errors, with importing the Avalonia the build is not able to infer all the required dlls needed from Avalonia
![image](https://user-images.githubusercontent.com/20876378/217787561-557785ca-5b62-40ac-9047-ff71912f4b02.png)